android/source/VideoLocker/res/drawable/switch_track.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@drawable/switch_defaultbg" android:state_focused="true"/>
+    <item android:drawable="@drawable/switch_cyanbg" android:state_checked="true"/>
     <item android:drawable="@drawable/switch_defaultbg" android:state_checked="false"/>
     <item android:drawable="@drawable/switch_cyanbg"/>
```

Comment:
It is not compulsory to add <code>state_checked="false"</code> as default drawable is defined. It was just to avoid any unwanted crashes if it does not read the default value for checked state. Hence I have added that line.